tests/test_utils.py

```diff
@@ -233,7 +233,7 @@ def test_get_alias_dict():
 
     assert alias_regression_a["c(rt, response)"]["c(rt, response)"] == "rt,response"
     assert alias_regression_a["c(rt, response)"]["Intercept"] == "v"
```

Comment:
Is this intended as `==` 'v' ?
Looks like you were expecting `==` 'a' ?

Reply:
This is actually correct. Since `v` is the `parent`, the `Intercept` for `rt, response` should be `v`, even though `a` is the parameter that has a regression, because the main formula for the model is `c(rt, response) ~ 1`

Reply:
ok makes sense thanks